packages/web-runtime/src/components/ApplicationsMenu.vue

```diff
@@ -1,14 +1,13 @@
 <template>
-  <div>
+  <nav aria-label>
     <oc-button
       id="_appSwitcherButton"
       ref="menubutton"
       appearance="raw"
       class="oc-topbar-menu-burger"
       :aria-label="applicationSwitcherLabel"
-      :uk-tooltip="applicationSwitcherLabel"
     >
-      <oc-icon name="apps" class="uk-flex" />
+      <oc-icon :uk-tooltip="applicationSwitcherLabel" name="apps" class="uk-flex" />
```

Comment:
Moved uk-tooltip to the icon because it overwrites the aria expand of the button